File: C++ Programming/problems on file handling/problem4.cpp
/*
4. Write application which accept file name from user and display size of file. 
	
	Input :  Demo.txt 
	
	Output :  File size is 56 bytes 
	
	Author : Milind Nalawade

	Date : 31/08/2021

	Language : C++ programming

*/

using namespace std;
#include <stdio.h>
#include<iostream>
// Entry point function
int main()
{
	char arr[30]={'\0'};
	FILE* fd;
	cout<<"\n Enter file name : ";
	cin.getline(arr,30);
	fd=fopen(arr,"r");
	if(fd==NULL)
	{
		cout<<"\n Unable to open the file \n";
	}
	fseek(fd,0L,SEEK_END);
	int no =ftell(fd);
	cout<<"\n Size of the file "<< arr<<" is "<<no<<" bytes \n";
	fclose(fd);
	return 0;
}

File: Java programming/problems on file handling/problem4.java
/*
4. Write application which accept file name from user and display size of file. 
	
	Input :  Demo.txt 
	
	Output :  File size is 56 bytes 
	
	Author : Milind Nalawade

	Date : 31/08/2021

	Language :Java 14.0.2

*/

import java.lang.*;
import java.util.*;
import java.io.File;
class problem4
{
	public static void main(String args[])
	{
		Scanner sobj=new Scanner(System.in);
		System.out.print("\n Enter file name : ");
		String str=sobj.nextLine();
		File fd=new File(str);
		if(fd==null)
		{
			System.out.print("\n Unable to open file "+str+"\n");
		}
		long no=fd.length();
		System.out.print("\n Size of file "+str+" is "+no+" bytes \n");
	}
}